layouts/partials/articles/main.njk

```diff
@@ -0,0 +1,27 @@
+<article class="grav-o-container grav-c-article">
+  <meta property="article:published_time" content="{{ date | safe }}">
+  {% if author.name %}<meta property="article:author" content="{{ author.name }}">{% endif %}
```

Comment:
Ok, according to [the spec](https://schema.org/Article), the value of an `schema:Article`'s `schema:name` property should be either a `schema:Person` or a `schema:Organisation`, so a simple string value of the author's name won't suffice.

Besides, _if_ we had actual author names, then we ought to display them too. I know we don't have designs for this yet, but from a markup point of view it could be something like:

```html
<span property="schema:author" typeof="schema:Person">
  <a property="schema:url" href="http://link.to/author/profile">
    <span property="schema:name">Joe Bloggs</span>
  </a>
</span>
```

However, a bit of testing in Google's [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool/u/0/) reveals that they consider certain properties mandatory. Their [docs](https://developers.google.com/search/docs/data-types/article#non-amp) seem to back this up.

Since, at least for now, the stories on the Buildit website won't show individual authors, we can instead set the Buildit organisation itself as the "author".

Google also insist on having setting a publisher, so let's point that to Buildit too.

Now, luckily, we already describe a Buildit `schema:Organisation` resource in the page header (well, the current v4 site does, so we should make sure we carry that over to v5 too) and, furthermore, it assigns it a URI: `https://buildit.wiprodigital.com/thing/buildit-org` (which is documented here: https://github.com/buildit/buildit/blob/develop/docs/metadata.md#resoure-uris

So, putting it all together, I'd swap this line with this link tags (note that it has 2 space separated property names):
```html
<link rel="schema:author schema:publisher" href="https://buildit.wiprodigital.com/thing/buildit-org" />
```

Similar to the image below, this is a `link` because the value is a resource, not a string value.

Reply:
I mocked up all my RDFa corrections inside Google's tool and it seems to work (I also ran it though the W3C RDFa validator and it checks out):

<img width="1318" alt="image" src="https://user-images.githubusercontent.com/16718916/58338253-5411c180-7e3f-11e9-8488-2a42f1493e08.png">
(ignore the errors about author and publisher not having types - with our complete header on the page, they will do)